doc/examples/segmentation/plot_chanvese.py

```diff
@@ -0,0 +1,25 @@
+""" This is an example to illustrate the us of the Chan-Vese Algorithm.
```

Comment:
Please, give a more detailed description including: main idea of the algorithm / difference from other segmentation algorithms already presented in `skimage`. Also, couple of reference links would be nice.